### PR TITLE
CNV-31216: Remove reset button from customize VM flow

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

Removing the reset button according to this [UX doc](https://docs.google.com/document/d/1IYkPyMu71UXtlvUR5C7btKSymo5E60GV_XYqCbfXLtI/edit)

## 🎥 Demo

Before:
remove-reset-b4
wrong-modal-title

After:
remove-reset
wrong-modal-title-after

### CODE REVIEW

File: src/views/catalog/wizard/tabs/overview/components/VMNameModal/VMNameModal.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const VMNameModal: React.FC<HostnameModalProps> = ({ isOpen, onClose, onSubmit,
 
   return (
     <TabModal
-      headerText={t('Edit hostname')}
```
Comment (hstastna) [inline src/views/catalog/wizard/tabs/overview/components/VMNameModal/VMNameModal.tsx:30]: Hm, this change seems to me unrelated to https://issues.redhat.com//browse/CNV-31216 or to the UX doc you've mentioned in the description of the PR. Also I cannot see this change in the attached screenshots. So I am confused. Is this change intentional?

**Review by hstastna — COMMENTED** [review]
Just one question, otherwise 👍🏽

File: src/views/catalog/wizard/tabs/overview/components/VMNameModal/VMNameModal.tsx (hunk 1 above)
Comment (avivtur) [reply src/views/catalog/wizard/tabs/overview/components/VMNameModal/VMNameModal.tsx:30]: they do relate, in previous PR - https://github.com/kubevirt-ui/kubevirt-plugin/pull/1428 
I add a modal to edit the name on the review section as part of the solution discussed with UX and PM.
So yes this change is itentional.
and added missing screenshots

**Review by hstastna — APPROVED** [review]
